.github/workflows/release-publish.yml

```
@@ -92,6 +92,9 @@ jobs:
           sky -c
           sky check
           sky show-gpus
+          sky dashboard
+          # Verify dashboard endpoint is accessible
+          curl -f http://127.0.0.1:46580/dashboard/ > /dev/null 2>&1 || echo "Dashboard endpoint is not accessible" && exit 1
```

```suggestion
          curl --fail --silent --show-error --output /dev/null http://127.0.0.1:46580/dashboard/ || echo "Dashboard endpoint is not accessible" && exit 1
```
This way if there is an error we can still see it.
Applies to the nightly build as well.